fw-child/resources/scss/_template.scss

```diff
@@ -104,6 +98,24 @@
   pointer-events: none;
 }
 
+.header-fixed-menu {
+  &.on-white {
+    #header-menu-icon {
+      color: $black;
```

Comment:
A dark grey color (instead of black) would be more in sync with the site's color.

Suggestion: `#666` but you can try another color.